tasks/configure.yml

```diff
@@ -1,8 +1,6 @@
 ---
-
 - name: Prometheus ElasticSearch Exporter service start
-  become: yes
   service:
-    name: prometheus-elasticsearch-exporter
+    name: elasticsearch-exporter
     state: started
-  when: prometheus_elasticsearch_exporter.start_on_boot
+  when: elasticsearch_exporter_start_on_boot
```

Comment:
This shouldn't be `enabled` instead of `state`?

Reply:
You mean L5?

Reply:
L5 or the variable name

Reply:
moved the systemd definition into `configure.yml` replacing the service definition in there